docs/pages/pmd/devdocs/metrics_howto.md

```diff
@@ -118,6 +119,62 @@ Notice that **we use an operation metric and a class node**. The `ResultOption`
 computed: you can choose among `HIGHEST`, `SUM` and `AVERAGE`. You can use metric options together with a result 
 option too.
 
+### Complete use case
+
+The following is a sample code for a rule reporting methods with a cyclomatic
+complexity over 10 and classes with a total cyclo over 50. A metric option can be
+user-configured with a rule property. More complete examples can be found in
+[CyclomaticComplexityRule]( https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/rule/CyclomaticComplexityRule.java)
+or [NcssCountRule]( https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/rule/NcssCountRule.java).
```

Comment:
Could you mention here either the DataClassRule or the GodClassRule as an example, too? These are the examples, that combine a couple of metrics. Thanks!